src/quantum/azext_quantum/_params.py

```diff
@@ -48,6 +66,7 @@ def load_arguments(self, _):
         c.argument('max_poll_wait_secs', max_poll_wait_secs_type)
 
     with self.argument_context('quantum job submit') as c:
+        c.argument('job_params', job_params_type)
```

Comment:
Do you have tests to cover this new argument?

Reply:
Following up offline on this. This argument is still not handled by the providers in our service, so there would be no effect on the tests recorded. There are tests on the next layer (our runtime) to that we exercise by calling the CLI, but we're only a pass through at this point.

Currently we can only exercise this path in our E2E suite, since we can't do job submissions without the Q# compiler/runtime and we haven't onboarded it to tests in this repo.

Reply:
In the last commit of the PR, the generation of args using this new parameter is being tested.